### PR TITLE
feat: [CDS-42907]: have single timezones static list

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

have single timezones static list

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11678)

File: src/modules/75-ce/common/TimeZoneSelector/TimeZoneSelector.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import React, { useState } from 'react'
 import { Virtuoso } from 'react-virtuoso'
 import { Popover, PopoverInteractionKind, Position } from '@blueprintjs/core'
 import { Container, Icon, TextInput, Text } from '@wings-software/uicore'
-import { ALL_TIME_ZONES } from '@ce/utils/momentUtils'
+import { ALL_TIME_ZONES } from '@common/utils/dateUtils'
```
Comment (ishantmehtaharness) [inline src/modules/75-ce/common/TimeZoneSelector/TimeZoneSelector.tsx:12]: this looks good

**Review by ishantmehtaharness — DISMISSED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by jenil-harness — APPROVED** [review]

**Comment by udayvunnam** [comment]
retrigger jest

**Review by mtran7 — APPROVED** [review]

**Comment by ishantmehtaharness** [comment]
retrigger jest

**Review by jyotiarora-harness — APPROVED** [review]

**Pull request merged**